### ISSUE TITLE
Compilation errors

### ISSUE THREAD

**Issue by 153957** [HiSPARC/infopakket#27]
There are numerous compilation errors when making gh-pages.
Some caused by the TeXLive 2015 update.
I have fixed some already, working on the test.

**Comment by 153957** [comment]
The Arduino documents (which use the `minted` package) do not compile with TeXLive 2014-2015.
Compilation does work with TeXLive 2013.

This is the error:

```
! Package tikz Error: Sorry, some package has redefined the meaning of the math
-mode dollar sign. This is incompatible with tikz and its calc library and migh
t cause unrecoverable errors.

See the tikz package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.12 ...{\PYZsh{}include \PYZlt{}BMP085.h\PYZgt{}}
```

The last bit is different for each minted env. I did try removing different lines of code, and sometimes it did compile, I have not found the exact cause. Using `\usepackage{minted1}` (available in TeXLive 2015 for backwards compatibility) did not help...

**Comment by 153957** [comment]
TLDR; Can be fixed by this: http://tex.stackexchange.com/a/165937/41702

I tried multiple ways of compiling the files. In the end I took the minted part from the artuino_weerstation.tex and put it in a separate file (keeping the same style). It also failed.
So I tried removing the second half of the code, perhaps it contained some bad character, it worked!
Then I tried removing the first half of the code (put the second half back), it also worked...
So perhaps it has to do with the length of the code. I tried splitting up the code in many short pieces; `\begin{minted}...\end{minted}...`. That sometimes worked.. The problem seems to be with the length of the code. So I removed the real code and simply replaced it by:
```latex
\begin{minted}{c}
a
a
…
a
\end{minted}
```
As long as the number of lines fit on the first page it compiled, as soon as it was one line to long it failed.. I did not find anything about not supporting page breaks (for minted env) in the minted documentation. Putting `\begin{minted}...\end{minted}` for each line of code is impractical and might mess up syntax highlighting. The problem seems to be related to TikZ and some other package that apparently redefines the math-mode, I have not yet tried to find out which is the culprit. This TeX stackexchange answer provides a 'solution': http://tex.stackexchange.com/a/165937/41702 . By making TikZ not fuss about the math-mode redefinition the file compiles correctly. The affected Arduino documents do not use TikZ except for the 'infopakket'-style. I did not see any strange effects in the compiled pdf. So this can be a temporary solution, it is not pretty and TikZ probably has a reason why it complains.
